packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1183,19 +1183,26 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   dynamic debugCreator;
 
   void _debugReportException(String method, dynamic exception, StackTrace stack) {
+    final RenderObject node = this;
     FlutterError.reportError(FlutterErrorDetailsForRendering(
       exception: exception,
       stack: stack,
       library: 'rendering library',
       context: ErrorDescription('during $method()'),
-      renderObject: this,
+      renderObject: node,
       informationCollector: () sync* {
         yield describeForError('The following RenderObject was being processed when the exception was fired');
         // TODO(jacobr): this error message has a code smell. Consider whether
         // displaying the truncated children is really useful for command line
         // users. Inspector users can see the full tree by clicking on the
         // render object so this may not be that useful.
         yield describeForError('This RenderObject', style: DiagnosticsTreeStyle.truncateChildren);
+        if (node.debugCreator?.element != null) {
+          yield node.debugCreator?.element?.describeOwnershipChain('Elements chain');
```

Comment:
you could write debugCreator?.element?.describeOwnershipChain
instead of 
node.debugCreator.

Reply:
fixed